frontend/src/styles/App.scss

```diff
@@ -41,3 +49,14 @@ body {
 .details:hover {
   color: #0000ff;
 }
+
+.ether-price {
+
+  p {
+    display: inline-block;
```

Comment:
Lets not overwrite all `<p>` elements to be inline-block. I'm not sure why we would want to do this, as this goes against the nature of `<p>` elements.

If there is some specific element you want to be an inline block, you can add the bootstrap inline-block class to it.